cognite/client/_api/extractionpipelines.py

```diff
@@ -315,6 +317,79 @@ def create(
         utils._auxiliary.assert_type(run, "run", [ExtractionPipelineRun, Sequence])
         return self._create_multiple(list_cls=ExtractionPipelineRunList, resource_cls=ExtractionPipelineRun, items=run)
 
+    def filter(
+        self,
+        external_id: str,
+        status: Literal["success", "failure", "seen"] | Sequence[Literal["success", "failure", "seen"]] | None = None,
+        created_time: TimestampRange | Dict[str, Any] | str | None = None,
+        message: str | None = None,
+        limit: int = LIST_LIMIT_DEFAULT,
+    ) -> ExtractionPipelineRunList:
+        """`Filter extraction pipeline runs <https://developer.cognite.com/api#tag/Extraction-Pipelines-Runs/operation/filterRuns>`_
```

Comment:
It's hard to see exactly how this is different from `def list(` on line 227, which according to the docstring also targets `Extraction-Pipelines/operation/filterRuns` 🤔